### PR TITLE
Try to get client IP from header 'x-forwarded-for'

### PR DESCRIPTION
**Pull request by vparfonov**
### What does this PR do?
fix for https://github.com/eclipse/che-theia/pull/624

### What issues does this PR fix or reference?
eclipse/che#15624

### CODE REVIEW

File: extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts [hunk 1]
```diff
@@ -18,9 +18,8 @@ export class CheClientIpServiceContribution implements BackendApplicationContrib
     configure(app: express.Application): void {
         app.get('/che/client-ip', (req, res) => {
             res.json({
-                ip: req.connection.remoteAddress,
-                ipFamily: req.connection.remoteFamily,
-                port: req.connection.remotePort
+                ip: req.get('x-forwarded-for') !== undefined ? req.get('x-forwarded-for') : req.connection.remoteAddress,
+                port: req.get('x-forwarded-port') !== undefined ? req.get('x-forwarded-port') : req.connection.remoteAddress,
```
Comment (RomanNikitenko) [inline extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts]: `: req.connection.remoteAddress` is it correct?

Comment (benoitf) [inline extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts]: @vparfonov please fix thx

**Comment by rhopp** [comment]
[ci-build]

File: extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts (hunk 1 above)
Comment (vparfonov) [reply extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts]: thanks

File: extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts [hunk 2]
```diff
@@ -18,9 +18,8 @@ export class CheClientIpServiceContribution implements BackendApplicationContrib
     configure(app: express.Application): void {
         app.get('/che/client-ip', (req, res) => {
             res.json({
-                ip: req.connection.remoteAddress,
-                ipFamily: req.connection.remoteFamily,
-                port: req.connection.remotePort
+                ip: req.get('x-forwarded-for') !== undefined ? req.get('x-forwarded-for') : req.connection.remoteAddress,
+                port: req.get('x-forwarded-port') !== undefined ? req.get('x-forwarded-port') : req.connection.remotePort,
```
Comment (azatsarynnyy) [inline extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts]: can be a bit shorter, but up to you :wink: 
```js
ip: req.get('x-forwarded-for') || req.connection.remoteAddress,
port: req.get('x-forwarded-port') || req.connection.remotePort
```

Comment (vparfonov) [reply extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts]: Thanks, but
Keep it simple, stupid

**Pull request merged**